src/IO/VarInt.h

```diff
@@ -198,10 +191,17 @@ inline const char * readVarUInt(UInt64 & x, const char * istr, size_t size)
     return istr;
 }
 
+[[noreturn]] inline void throwValueTooLargeForVarIntEncodingException(UInt64 x)
+{
+    throw Exception(ErrorCodes::BAD_ARGUMENTS, "Value {} is too large for VarInt encoding", x);
```

Comment:
Maybe it worth to put a comment that `LOGICAL_ERROR` (as it should) is not used here by intention?

Reply:
Tnx, that makes sense.